### ISSUE TITLE
Ingress support for Elasticsearch and Kibana CRs

### ISSUE THREAD

**Issue by gbaso** [elastic/cloud-on-k8s#8079]
After #7941 I expected to also be able to add the ingress to my es/kibana definitions when using the ECK operator. However I cannot find any reference to `ingress` in the [CRDs](https://github.com/elastic/cloud-on-k8s/blob/main/config/crds/v1/all-crds.yaml).

Is the ingress configuration via the ECK operator not supported?

**Comment by thbkrkr** [comment]
Ingress configuration is not supported via the ECK operator CRDs. The changes in #7941 added Ingress support in the Helm charts for Kibana and Elasticsearch. You'll need to configure Ingress separately using standard Kubernetes resources if you do not use the Helm charts.

Documentation: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-stack-helm-chart.html#k8s-eck-stack-ingress

**Comment by barkbay** [comment]
Closing a duplicate of https://github.com/elastic/cloud-on-k8s/issues/7008
